[Issue 8364][docs] Update decommission content in different releases (#8368)
apache/pulsar: site2/docs/administration-zk-bk.md

> ### Decommissioning bookies cleanly

Reviewer: ```suggestion
### Decommission bookies cleanly
```

> When you want to decommission a bookie, the following process is useful to follow in order to verify if the decommissioning was safely done.

Reviewer: To decommission a bookie safely, follow the following process.

> #### Before you decommission

Reviewer: ```suggestion
#### Before decommission
```

> 1. Ensure state of your cluster can support the decommissioning of the target bookie.

Reviewer: ```suggestion
1. Ensure the state of your cluster supports decommissioning the target bookie.
```

Check if `EnsembleSize >= Write Quorum >= Ack Quorum` stays true with one less bookie

> 2. Ensure target bookie shows up in the `listbookies` command.

Reviewer: ```suggestion
2. Ensure the target bookie shows up in the `listbookies` command.
```

3. Run the decommission command.
If you have logged onto the node you wish to decommission, you don't need to provide `-bookieid`

> If you are running the decommission command for target bookie node from another bookie node you should mention the target bookie id in the arguments for `-bookieid`

Reviewer: ```suggestion
If you are running the decommission command for the target bookie node from another bookie node, you should mention the target bookie ID in the arguments for `-bookieid`
```

`$ bin/bookkeeper shell decommissionbookie`
or
`$ bin/bookkeeper shell decommissionbookie -bookieid <target bookieid>`

> 4. Validate that no ledgers are on decommissioned bookie.

Reviewer: ```suggestion
4. Validate that no ledgers are on the decommissioned bookie.
```
